app/javascript/blacklight/core.js

```diff
@@ -19,21 +19,23 @@ Blacklight = function() {
           listeners.push('turbolinks:load');
         } else {
           // Turbolinks < 5
-          listeners.push('page:load', 'ready');
+          listeners.push('page:load', 'DOMContentLoaded');
         }
       } else {
-        listeners.push('ready');
+        listeners.push('DOMContentLoaded');
       }
 
-      return listeners.join(' ');
+      return listeners;
     }
   };
 }();
 
 // turbolinks triggers page:load events on page transition
 // If app isn't using turbolinks, this event will never be triggered, no prob.
-$(document).on(Blacklight.listeners(), function() {
-  Blacklight.activate();
-});
+Blacklight.listeners().forEach(function(listener) {
```

Comment:
Is the change here also related?

Reply:
Yes, I changed `Blacklight.listeners()` to return an array of values that `document.addEventListener` can accept.

Reply:
OK, sounds good.